src/lib/api/builder.ts

```diff
@@ -314,7 +315,8 @@ function toRemoteItem(item: Item): RemoteItem {
     contents: item.contents,
     content_hash: item.contentHash,
     created_at: new Date(item.createdAt),
-    updated_at: new Date(item.updatedAt)
+    updated_at: new Date(item.updatedAt),
+    local_content_hash: item.serverContentHash
```

Comment:
maybe we want to move this prop (same [here](https://github.com/decentraland/builder/pull/1820/files#diff-9b32bce51ccd6629860aa36212c39f1a85965c2dccd90c6ec0d6c5d7bb8d0159R342)) below content_hash so it's a bit more "content aware"

Reply:
Moved it!